client/app/controllers/map.js

```diff
@@ -13,7 +15,14 @@ export default Ember.Controller.extend({
   schools: Ember.computed.mapBy('assignedStudents', 'school'),
   uniqueSchools: Ember.computed.uniq('schools'),
 
-  showingAllStudents: Ember.computed('students.@each.isLocated', function() {
-    return this.get('students').every((student) => student.get('isLocated'));
-  })
+  showingAllStudents: Ember.computed('dismissAllStudentsAlert', 'students.@each.isLocated', function() {
```

Comment:
The name `showingAllStudents` here is no longer accurate, since the value could be true even if not all students are actually being shown, due to the "dismiss" override. Maybe the logic could be inverted and this could be named `studentsAlertActive`, since that would correspond better with its real purpose anyway.